openedx/core/djangoapps/signals/signals.py

```diff
@@ -11,3 +11,11 @@
 # TODO: runtime coupling between apps will be reduced if this event is changed to carry a username
 # rather than a User object; however, this will require changes to the milestones and badges APIs
 COURSE_CERT_AWARDED = Signal(providing_args=["user", "course_key", "mode", "status"])
+
+# Signal that indicates that a user has passed a course.
+COURSE_GRADE_NOW_PASSED = Signal(
```

Comment:
@bderusha @amangano-edx FYI. I think we'll want to use this signal to determine when the task that makes requests to credentials is enqueued.